pkg/cosesign1/misc.go

```diff
@@ -0,0 +1,159 @@
+package cosesign1
+
+/*
+	Little handy utilities that make logging and a command line tool easier.
+*/
+
+import (
+	"fmt"
+	"os"
+
+	"crypto/x509"
+	"encoding/base64"
+	"io"
+	"log"
+
+	"github.com/veraison/go-cose"
+)
+
+func ReadBlob(path string) []byte {
+	content, err := os.ReadFile(path)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return content
+}
+
+func ReadString(path string) string {
+	content := ReadBlob(path)
+	str := string(content[:])
+	return str
+}
+
+// to replace the rand.Reader with a fixed salt to make debugging easier.
+
+type fixedReader struct {
+	valueToReturn byte
+}
+
+func (fr *fixedReader) Read(p []byte) (int, error) {
+	if len(p) > 0 {
+		p[0] = fr.valueToReturn
+		return 1, nil
+	}
+	return 0, nil
+}
+
+func NewFixedReader(value byte) io.Reader {
+	return &fixedReader{valueToReturn: value}
+}
+
+func WriteBlob(path string, data []byte) error {
+	return os.WriteFile(path, data, 0644)
+}
+
+func WriteString(path string, str string) error {
+	var data = []byte(str)
+	return WriteBlob(path, data)
+}
+
+func x509ToBase64(cert *x509.Certificate) string {
+	base64Cert := base64.StdEncoding.EncodeToString(cert.Raw)
+
+	return base64Cert
+}
+
+func keyToBase64(key any) string {
+
```

Comment:
extra newline

Reply:
I guess you mean line 68, after the function signature line? Also, several others in this file. Can the linter flag these?